terminal/print.go

```diff
@@ -5,21 +5,21 @@ import (
 )
 
 var (
-	ansiStdout = NewAnsiStdout()
+	AnsiStdout = NewAnsiStdout()
```

Comment:
okay, np, the naming is pretty arbitrary to me.  Really we should probably replace all the `fmt.Print*(...)` calls with `fmt.Fprint*(Stdout, ...)` so we can test/capture all the output, but that is probably a change for later.